source/reference/collection-level-filtering.txt

```diff
@@ -0,0 +1,203 @@
+.. _c2c-collection-filtering:
+
+==========================
+Collection Level Filtering
+==========================
+
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+.. versionadded:: 1.1
+
+{+c2c-product-name+} provides continuous data synchronization or a 
+one-time data migration between two MongoDB clusters. You can use
+collection level filtering to control the data that the :ref:`mongosync
+<c2c-mongosync>` utility transfers between the source and destination
+clusters.
+
+.. _c2c-configure-filter:
+
+Configure a Filter
+------------------
+
+.. important::
+
+   Once you start ``mongosync`` with a filter in place, the filter
+   cannot be modified directly. If you do need to modify the filter,
+   see: :ref:`c2c-change-filter`.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Identify Databases and Collections.
+
+      Identify the databases and collections that you want to sync to
+      the destination cluster. When you add a set of databases to the
+      filter, you also exclude any other databases in the cluster.
+      
+      If specify a collection in your filter, you also exclude any other
+      collections that are in the same database.
+
+   .. step:: Create a Filter.
+
+      The ``includeNamespaces`` option specifies a filter that you can
+      pass to the :ref:`/start <c2c-api-start>` API. 
+
+      The filter syntax is: 
+
+      .. include:: /includes/api/facts/includeNamespaces-syntax.rst
+
+      Create an entry in the ``includeNamespaces`` array for each
+      database that you identified in step 1. Use the ``"database"``
+      field to specify the database name.
+      
+      If you want to filter on collections within a database, add those
+      collections to the ``"collections"`` field for that database
+      entry.
+
+   .. step:: Use the Filter.
+
+      To use the filter, attach the filter document when you make the
+      :ref:`/start <c2c-api-start>` API call to begin syncing.
+
+      .. literalinclude:: /includes/api/requests/start-filtered.sh
+         :language: shell
+
+For an example configuration, see: :ref:`c2c-example-start-with-filter`.
+
+.. _c2c-change-filter:
+
+Change the Filter
+-----------------
+
+You cannot update an existing filter. You must remove the old filter and
+restart ``mongosync`` with a new one. 
+
+When ``mongosync`` ran your original filter, it created user databases
+and the ``mongosync_reserved_for_internal_use`` system database on the
+destination cluster. You must remove those databases before restarting
+``mongosync`` with your new filter.
+
+Follow these steps to prepare the destination cluster for a new filter.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Remove ``mongosync_reserved_for_internal_use``.
+
+      a. Stop the ``mongosync`` process. 
+      
+      #. Connect to the :binary:`mongos` instance in the destination
```

Comment:
Filtered sync also works on replica sets, can you add that you can connect to the primary mongod